tasks/__init__.py

```diff
@@ -520,6 +520,7 @@ def test(ctx, all=False, syntax=False):
         flake(ctx)
         jshint(ctx)
 
+    test_js(ctx)
```

Comment:
driveby: Should this be a new build or added to our current fasted? Probably can be added to api1 (fastest) and will still be faster than our longest. @sloria @brianjgeiger